frontend/src/App.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import { BrowserRouter as Router, Route } from 'react-router-dom';
+import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
```

Comment:
cleanup: Are we still using `Link`?

Reply:
Ok -- this is a rebase issue, #39 removed `Link`

Reply:
Got it - when I did the resolved the rebase conflict, I must've chose my version of this hunk on accident. Will fix.